editor/plugins/script_text_editor.cpp

```diff
@@ -802,7 +802,7 @@ void ScriptEditor::_update_modified_scripts_for_external_editor(Ref<Script> p_fo
 	}
 
 	for (const Ref<Script> &E : scripts) {
-		Ref<Script> scr = E;
+		const Ref<Script> &scr = E;
```

Comment:
All cases like this can be removed, as there's no need to convert a `const Ref<T>` to `Ref<T>` as pointer access on a `Ref<T>` are agnostic to that:
```cpp
_FORCE_INLINE_ T *operator->() const {
	return reference;
}
```